app/views/OfferedCourses/Courses.js

```diff
@@ -14,22 +14,22 @@ class Courses extends Component {
     }
     async componentDidMount() {
         const courses = await ajax.fetchInitialCourses();
-        this.setState(prevState => {
-            return { courses };
-        });
-        console.log(courses);
+        this.setState({ courses });
+        console.log('COURSES' + courses);
     }
 
 
     render() {
 
-        //console.log(this.state.courses);
-        //console.log(Object.keys(this.state.courses).length);
+        console.log('STATE COURSES ');
+        console.log(this.state.courses.data);
+        console.log('COURSES LENGTH' + Object.keys(this.state.courses).length);
+        console.log('COURSES LENGTH' + Object.keys(this.state.courses).length);
```

Comment:
I think that we shouldn't merge to master with debugging comments.

Reply:
The code will be cleaned up when I’m done getting all the data we need from the api. However, merging at this stage will help us move forward